samples/cpp/benchmark_app/benchmark_app.hpp

```diff
@@ -402,48 +404,48 @@ static void show_usage() {
     std::cout << "benchmark_app [OPTION]" << std::endl;
     std::cout << "Options:" << std::endl;
     std::cout << std::endl;
-    std::cout << "    -h, --help                " << help_message << std::endl;
-    std::cout << "    -m \"<path>\"               " << model_message << std::endl;
-    std::cout << "    -i \"<path>\"               " << input_message << std::endl;
-    std::cout << "    -d \"<device>\"             " << target_device_message << std::endl;
-    std::cout << "    -extensions \"<absolute_path>\" " << custom_extensions_library_message << std::endl;
-    std::cout << "    -c \"<absolute_path>\"      " << custom_cldnn_message << std::endl;
-    std::cout << "    -hint \"performance hint (latency or throughput or cumulative_throughput or none)\"   "
+    std::cout << "    -h, --help                    " << help_message << std::endl;
+    std::cout << "    -m  <path>                    " << model_message << std::endl;
+    std::cout << "    -i  <path>                    " << input_message << std::endl;
```

Comment:
Just curious, why you put two spaces between `i` and `<path>`?

Reply:
This is just for vertical alignment between nearby lines. It doesn't mean that 2 spaces are actually required.